<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-BLN1FMN" failures="1" tests="1" name="SoftError" time="0.036" errors="0" timestamp="2020-05-04T18:58:15 IST">
  <testcase classname="com.selenium.testng.SoftvsHardErrorTest" name="softError" time="0.02">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [true] but found [false],
	expected [false] but found [true],
	expected [Failed] but found [Sample]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false],
expected [false] but found [true],
expected [Failed] but found [Sample]
at com.selenium.testng.SoftvsHardErrorTest.softError(SoftvsHardErrorTest.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- softError -->
  <testcase classname="com.selenium.testng.SoftvsHardErrorTest" name="hardError" time="0.0">
    <ignored/>
  </testcase> <!-- hardError -->
</testsuite> <!-- SoftError -->
